OwenConferenciasApp/screens/NewEvents.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import {
+  View,
+  Text,
+  TouchableOpacity,
+  StyleSheet,
+  ScrollView,
+  TextInput,
+  Alert,
+} from 'react-native';
+
+export default class NewEvent extends React.Component {
+  render() {
+    return (
+      <View style={styles.mainContainer}>
+        <ScrollView style={styles.scrollView}>
+          <View style={styles.cardEvent}>
+            <Text style={styles.txtTitle}>New conference</Text>
+            <View style={styles.formNewEvent}>
+              <TextInput
+                style={styles.inputEvents}
+                placeholder="Name of the conference:"
+              />
+              <TextInput style={styles.inputEvents} placeholder="Topic:" />
+              <TouchableOpacity
+                style={styles.btnDayHour}
+                onPress={this.showPicker}>
+                <Text style={styles.txtButtons}>Select the day</Text>
+              </TouchableOpacity>
+              <TouchableOpacity style={styles.btnDayHour}>
+                <Text style={styles.txtButtons}>Select the hour</Text>
+              </TouchableOpacity>
+              <TouchableOpacity
+                style={styles.btnAddConf}
+                onPress={() =>
+                  Alert.alert('Are you sure ta add this conference')
+                }>
+                <Text style={styles.txtButtons}>Add conference</Text>
+              </TouchableOpacity>
+            </View>
+          </View>
+        </ScrollView>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  mainContainer: {
+    height: '100%',
```

Comment:
I recommend to avoid the use of percentages, and prioritize the use of flex tools
https://reactnative.dev/docs/flexbox

Reply:
same apply for all components where you use percentages